test-utils/state-viewer/Cargo.toml

```diff
@@ -9,6 +9,8 @@ ansi_term = "0.12"
 borsh = "0.9"
 clap = "2.33"
 tracing = "0.1"
+indicatif = {version = "0.16.2", features = ["rayon"]}
```

Comment:
TBH, not a huge fun indicatif for *this* use case. 

* It can be slow if used naively. For me, `genesis-populate`  takes 7 seconds with a progress bar and 6 without. That's unreasonably high cost for a UI-nicety. 
* It has (had?) some funky buffering inside, which eat printfs and panic messages: https://github.com/mitsuhiko/indicatif/issues/121. 
* for non-polished dev-tools, to much output is a significantly bigger problem than too little output. Its best to stick to few log lines if possible. 


Where indicatiff shines is for building delightful tools for the end user, where you think very carefully about the overall output and judiciously place just the right amount of prints. For internal dev tooling, less is more.

Reply:
To clarify, that's my personal opinion, and not a very strong one! I am fine with keeping it!